lib/create/index.js

```diff
@@ -0,0 +1,73 @@
+const {
+  GitHubUser,
+  Subscription,
+} = require('../models');
+
+const createIssueDialog = require('../messages/create/create-issue-dialog');
+
+async function openDialog(req, res) {
+  const {
+    command,
+    gitHubUser,
+    slackWorkspace,
+  } = res.locals;
+
+  const subscriptions = await Subscription.findAll({
+    where: { channelId: req.body.channel_id, slackWorkspaceId: slackWorkspace.id },
+  });
+  if (subscriptions.length === 0) {
+    return command.respond({ text: 'This channel is not subscribed to any repositories' });
+  }
```

Comment:
It'd be great if a user could also pass in an explicit repo here. `github open owner/repo`

Reply:
Do you think we could add that in a follow up PR?